huaweicloud/services/cts/resource_huaweicloud_cts_tracker.go

```diff
@@ -71,6 +71,24 @@ func ResourceCTSTracker() *schema.Resource {
 				Optional:     true,
 				RequiredWith: []string{"bucket_name"},
 			},
+			"compress_type": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				RequiredWith: []string{"bucket_name"},
+			},
+			"is_sort_by_service": {
+				Type:         schema.TypeBool,
+				Optional:     true,
+				RequiredWith: []string{"bucket_name"},
+				Default:      true,
+			},
+			"exclude_service": {
+				Type:     schema.TypeList,
```

Comment:
```suggestion
				Type:     schema.TypeSet,
```
if more services are supported, the update method should not be traggered when the value order is modified

Reply:
fixed